index.tsx

```diff
@@ -124,7 +124,15 @@ function App() {
         // two reverse proxies.
         ADAPTABLE_TRUST_PROXY_DEPTH: "2",
     };
-    const env = mergeEnvSimple(dbEnv, standardEnv, config.environment);
+
+    // Remove PORT from the container environment
+    const { PORT, ...env } = mergeEnvSimple(dbEnv, standardEnv, config.environment) || {};
+
+    let port = 80;
+    if (PORT) {
+        const p = parseInt(PORT, 10);
+        if (!Number.isNaN(p)) port = p;
```

Comment:
parseInt will parse only leading characters.  So if PORT is "10monkeys" you still get a PORT of 10.  Do we care to fix this?